scripts/process.ts

```diff
@@ -1,14 +1,19 @@
 const preprocess = require('preprocess');
 
-import { existsSync, mkdirSync, write, writeFileSync } from 'fs';
+import { existsSync, mkdirSync, writeFileSync } from 'fs';
 import { SYSTEM_CONTRACTS, getRevertSelector, getTransactionUtils } from './constants';
 import * as hre from 'hardhat';
 import { ethers } from 'ethers';
 import { renderFile } from 'template-file';
 import { utils } from 'zksync-web3';
 import { ForceDeployment } from './utils';
+import { join } from "path";
+
 const OUTPUT_DIR = 'bootloader/build';
```

Comment:
I think we should remove `OUTPUT_DIR` and all its mentions for consistency.

Reply:
So, where do you think preprocessed bootloaders should be stored? I am interested because it touches on things on which I am working on

Reply:
@AntonD3 The same path as previously, this PR only changes the script how path is derived.